advisories/github-reviewed/2025/03/GHSA-jr5f-v2jv-69x6/GHSA-jr5f-v2jv-69x6.json

```diff
@@ -11,7 +11,7 @@
   "severity": [
     {
       "type": "CVSS_V4",
-      "score": "CVSS:4.0/AV:N/AC:L/AT:N/PR:N/UI:N/VC:H/VI:N/VA:N/SC:N/SI:N/SA:N/E:P"
```

Comment:
I didn't intend to remove the `E` metric, but the advisory improvement screen forced me to remove it, and I don't see a way to put it back.

Reply:
I've opened an issue about this limitation in the advisory improvement screen: https://github.com/github/advisory-database/issues/5357.